CliClient/tests/test-utils.js

```diff
@@ -439,6 +439,7 @@ async function createNTestFolders(n) {
 	for (let i = 0; i < n; i++) {
 		let folder = await Folder.save({ title: 'folder' });
 		folders.push(folder);
+		await time.msleep(10);
```

Comment:
Just to ensure the created items have different timestamps.  Especially with notes, this can affect the sort order if the timestamps are the same, and so can cause random test failures.

Having identical timestamps is fine in the app, just not so good for testing.